src/msg/consumer/handlers_test.go

```diff
@@ -112,33 +116,35 @@ func TestServerWithConsumeFn(t *testing.T) {
 		closed = true
 	}
 
-	l, err := net.Listen("tcp", "127.0.0.1:0")
-	require.NoError(t, err)
+	for _, opts := range testOpts {
+		l, err := net.Listen("tcp", "127.0.0.1:0")
+		require.NoError(t, err)
 
-	// Set a large ack buffer size to make sure the background go routine
-	// can flush it.
-	opts := testOptions().SetAckBufferSize(100)
-	s := server.NewServer("a", NewConsumerHandler(consumeFn, opts), server.NewOptions())
-	require.NoError(t, err)
-	s.Serve(l)
+		// Set a large ack buffer size to make sure the background go routine
```

Comment:
tabs? go fmt :-)

Reply:
Hmm... I think my go fmt is using tabs. Not sure if that is the default, but if I run it all spaces covert to tabs.

Reply:
Right, too much Python recently ;)